storybook/stories/dash/Users/EditUser.stories.tsx

```diff
@@ -47,11 +47,92 @@ export const Default: Story = props({
   addNewBlockedApp: () => {},
   removeBlockedApp: () => {},
   newBlockedAppIdentifier: ``,
+  blockedApps: [
+    { id: `1`, identifier: `Notepad++` },
+    {
+      id: `2`,
+      identifier: `Xcode`,
+      schedule: {
```

Comment:
add `import { defaults } from '@dash/types';` to the top and then you can get rid of this whole chunk if you're ok with the default one: `defaults.ruleSchedule()`